README

```diff
@@ -42,12 +42,15 @@
 
       Before installing the Guest VM ("--install" option used), you must complete the following manual tasks:
 
-        1) Place the ISO image for the guest OS in the isos/ folder.  The install option 
+        1) Place the ISO image for the guest OS in the `isos/` folder.  The install option
            currently supports only the Fedora 23 and 24 OS ISO & guest.
 
-           Link to the currently supported Guest VM Fedora ISO: 
-             >> http://archives.fedoraproject.org/pub/archive/fedora-secondary/releases/23/Server/ppc64le/iso/Fedora-Server-DVD-ppc64le-23.iso
-             >> http://archives.fedoraproject.org/pub/archive/fedora-secondary/releases/24/Server/ppc64le/iso/Fedora-Server-dvd-ppc64le-24-1.2.iso
+           Download Fedora images and place them at the expected paths:
+             >> curl -o isos/Fedora-Server-DVD-ppc64le-23.iso http://archives.fedoraproject.org/pub/archive/fedora-secondary/releases/23/Server/ppc64le/iso/Fedora-Server-DVD-ppc64le-23.iso
+             >> curl -o isos/Fedora-Server-DVD-ppc64le-24.iso http://archives.fedoraproject.org/pub/archive/fedora-secondary/releases/24/Server/ppc64le/iso/Fedora-Server-dvd-ppc64le-24-1.2.iso
```

Comment:
Is there a specific reason why you changed the files name from the original iso image name for Fedora 24?  We should ensure we are instructing users correctly and that the code is not looking for a specific ISO file name based on the guest VM OS choice.